code/controllers/subsystem/parallax.dm

```diff
@@ -4,11 +4,21 @@ SUBSYSTEM_DEF(parallax)
 	flags = SS_POST_FIRE_TIMING | SS_BACKGROUND
 	priority = FIRE_PRIORITY_PARALLAX
 	runlevels = RUNLEVEL_LOBBY | RUNLEVELS_DEFAULT
-	var/list/currentrun
+	var/current_run_pointer = 1
+	var/list/currentrun = list()
+	var/list/queued = list()
 	var/planet_x_offset = 128
 	var/planet_y_offset = 128
 	var/random_layer
 	var/random_parallax_color
+	//Amount of ticks between the parallax being allowed to freely fire without going into the queue
+	var/parallax_free_fire_delay_ticks = 10
+
+	//Check pop limits
+	var/throttle_ghosts = FALSE
+	var/throttle_all = FALSE
+	var/throttle_ghost_pop = 0
+	var/throttle_all_pop = 0
```

Comment:
isn't this a bit overkill? Runtime should be fuck all compared to other systems. Plus throttling this will actually have effects on player experience

Reply:
It's barely noticable if its disabled, since the background still moves, it just moves at the same speed as the player.
Performance of parallax at high pops is extremely poor, hence the creation of this PR in the first place.
![image](https://user-images.githubusercontent.com/26465327/179737531-955afa7e-f951-4ef2-9f59-2274052a133f.png)
This is from a round with 64 players.

Reply:
https://user-images.githubusercontent.com/26465327/179738920-92c9ea5d-cad4-47d8-bd71-463702e7c585.mp4

Here is a video of me flying around with parallax completely disabled. In addition to this, shuttle parallax still works.
If you don't want it, then you can disable it via the config settings (It comes disabled by default).

This setting is intended for servers with populations >70/80.